README.md

```diff
@@ -162,6 +162,28 @@ $ manalize setup --no-update ~/my-awesome-app
 
 So you can then apply a patch provided by the [diff](#diff) command.
 
+Contributing
+------------
+
+#### Getting the source code
+```sh
+$ git clone git@github.com:manala/manalize <path-to-manalize>
```

Comment:
What do you think about removing the `$`?

Reply:
👍